frameworks/g-cloud-9/questions/services/staffSecurityClearanceChecks.yml

```diff
@@ -10,8 +10,11 @@ depends:
 type: radios
 options:
   - label: Staff screening performed with conforms to BS7858:2012
+    value: "staff_screening_to_bs7858:2012"
   - label: Staff screening performed but doesn’t conform with BS7858:2012
+    value: "staff_screening_not_bs7858:2012"
```

Comment:
If we're avoiding `*nix` as a value maybe we could drop the colons from these?  I can't really think of an issue they will cause, but, you know...